### PR TITLE
feat: add multi-camera demo

### PR DESCRIPTION
**Pull request by JaimeTorrealba**
**Attention**: I needed to update the core version, that result in the portal example get broken for the particles problem

As always any feedback let me know

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -19,5 +19,6 @@
   "testing.automaticallyOpenPeekView": "never",
   "files.associations": {
     "*.mdx": "markdown"
-  }
+  },
+  "vue.features.codeActions.enable": false
```
Comment (alvarosabu) [inline .vscode/settings.json:23]: ?

File: components/content/MultiCameraDemo.vue [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  currentCam.viewport = data.viewPort
+  currentCam.position.set(...data.position)
+  currentCam.position.multiplyScalar(data.factor)
+  currentCam.lookAt(0, 0, 0)
+  currentCam.updateMatrixWorld()
+  cameras.push(currentCam)
+})
+
+const { scene: model, animations } = await useGLTF(
+  'https://raw.githubusercontent.com/Tresjs/assets/main/models/gltf/warcraft-3-alliance-footmanfanmade/source/Footman_RIG.glb',
```
Comment (alvarosabu) [inline components/content/MultiCameraDemo.vue:54]: @JaimeTorrealba forgot to ask you, what are the licenses for that model, is it [C0](https://creativecommons.org/share-your-work/public-domain/cc0/)?

**Review by alvarosabu — CHANGES_REQUESTED** [review]
I love the overall result, great demo

If you merge the last `main` now you can add to the `.md` a `TheInfo` component to add the code of the experiment in a markdown like this https://github.com/Tresjs/playground/blob/main/content/experiments/basic-shaders.md

File: .vscode/settings.json (hunk 1 above)
Comment (JaimeTorrealba) [reply .vscode/settings.json:23]: Done

File: components/content/MultiCameraDemo.vue (hunk 2 above)
Comment (JaimeTorrealba) [reply components/content/MultiCameraDemo.vue:54]: Yes it's free... 

you can check it out here: https://sketchfab.com/3d-models/warcraft-3-alliance-footmanfanmade-201452e568064aedadccfafb668ef6a5

File: .vscode/settings.json (hunk 1 above)
Comment (alvarosabu) [reply .vscode/settings.json:23]: I mean, what does this line do?

File: components/content/MultiCameraDemo.vue [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+onLoop(() => {
+  if (mixer) {
+    mixer.update(0.01)
+  }
+})
+</script>
+<template>
+  <TresCanvas window-size clear-color="#82DBC5" class="over-hidden">
+    <TresArrayCamera :args="[cameras]" :position="[0, 2, 5]" />
+    <Suspense>
```
Comment (alvarosabu) [inline components/content/MultiCameraDemo.vue:99]: Checking the deploy on Netlify, the model takes time to load and a blank page is sown that could be misleading.

Can you please add a loader? You can check the PostProcessingManual example

```ts
//TODO: replace this when UseProgress is implemented https://github.com/Tresjs/cientos/issues/22

const hasFinishLoading = ref(false)
const progress = ref(0)

let saveLastTotalLoaded = 0

DefaultLoadingManager.onProgress = (item, loaded, total) => {
  if (loaded === total) {
    saveLastTotalLoaded = total
    hasFinishLoading.value = true
  }
  progress.value = Math.round(((loaded - saveLastTotalLoaded) / (total - saveLastTotalLoaded)) * 100 || 100, 2)
}
```

```
 <Transition
    name="fade-overlay"
    enter-active-class="opacity-1 transition-opacity duration-200"
    leave-active-class="opacity-0 transition-opacity duration-200"
  >
    <div
      v-show="!hasFinishLoading"
      class="absolute bg-grey-600 t-0 l-0 w-full h-full z-20 flex justify-center items-center text-black font-mono"
    >
      <div class="w-200px">
        Loading... {{ progress }} %
        <span
          class="mt-4 block bg-black h-1 w-1 z-10 transition-1"
          :style="{ transform: `scaleX(${progress / 2})`, transformOrigin: '0 100%' }"
        ></span>
      </div>
    </div>
  </Transition>
```

Comment (JaimeTorrealba) [reply components/content/MultiCameraDemo.vue:99]: The implementation of the loader, using the useGltf seems like it's not working properly (or I don't know how to use it well)

I made the merge to last main

**Comment by JaimeTorrealba** [comment]
@alvarosabu Please check this out, 

I update this demo, and added useProgress()

**Pull request merged**